pages/Selectors/buildShipmentPage.js

```diff
@@ -0,0 +1,36 @@
+const BuildShipmentPage =  {
```

Comment:
You should never use Absolute xpath, change all the absolute XPath, and make them relative. Also playwright offers so wide and good set of locator selection options that could be used, why you haven't looked for the best approaches while selecting locators. Please make changes.